gui/src/gui_state_container.h

```diff
@@ -240,6 +243,7 @@ public slots:
   quint32 commanded_pip_ = 15;
   quint32 commanded_peep_ = 5;
   qreal commanded_i_time_ = 1.0;
+  qreal commanded_fio2_percent_ = 80.0;
```

Comment:
If this is the default, it's kinda weird for us rn, since we only plan to support 21% and 100%.

Reply:
Changed to 100